modules/acm/feature.tf

```diff
@@ -42,6 +42,7 @@ resource "google_gke_hub_feature_membership" "main" {
       for_each = var.enable_config_sync ? [{ enabled = true }] : []
 
       content {
+        enabled       = var.enable_config_sync
```

Comment:
Do we still need line 42 `for_each = var.enable_config_sync ? [{ enabled = true }] : []`?

Reply:
Yes, as this is currently a dynamic block and the block content only populated based on the for_each.  However the actual value of `[{ enabled = true }]` isn't important other than it evaluates to length `1`.